src/components/SideBar/ItemDrawer/ItemDrawer.tsx

```diff
@@ -40,17 +42,20 @@ export default class ItemDrawer extends React.PureComponent<Props, State> {
 
   render() {
     const { isList } = this.state
+    const { categories } = this.props
 
     return (
       <div className="ItemDrawer">
         <Header size="medium" className="title">
           {t('itemDrawer.title')}
         </Header>
-        <Drawer label="Category">
-          <Grid columns={1} padded="horizontally" className={`asset-grid ${isList ? 'item-list' : 'item-grid'}`}>
-            {this.renderGrid()}
-          </Grid>
-        </Drawer>
+        {categories.map((category, index) => (
+          <Drawer key={index} label={category.name}>
+            <Grid columns={1} padded="horizontally" className={`asset-grid ${isList ? 'item-list' : 'item-grid'}`}>
```

Comment:
should the columns prop be 3 on grid mode?

Reply:
Yes!